iothub/service/src/Twin/Models/ClientTwinJsonConverter.cs

```diff
@@ -1,422 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.Linq;
-using System.Reflection;
-using Azure;
-using System.Text.Json;
-using System.Text.Json.Serialization;
-
-namespace Microsoft.Azure.Devices
-{
-    /// <summary>
-    /// Converts a twin to JSON.
-    /// </summary>
-    internal sealed class ClientTwinJsonConverter : JsonConverter<ClientTwin>
```

Comment:
Nice that we can get rid of all this serialization/deserialization code.

Reply:
I hope so.